File: src/gestorAplicacion/Hija/TarjetaNegocioUsuario.java
package gestorAplicacion.Hija;

public class TarjetaNegocioUsuario {
    private String nombreNegocio;
    private String descripcionNegocio;
    private String telefonoNegocio;
    private String direccionNegocio;

    public TarjetaNegocioUsuario(String nombreNegocio, String descripcionNegocio, String telefonoNegocio, String direccionNegocio){
        this.nombreNegocio = nombreNegocio;
        this.descripcionNegocio = descripcionNegocio;
        this.telefonoNegocio = telefonoNegocio;
        this.direccionNegocio = direccionNegocio;
    }

    public String getNombreNegocio() {
        return nombreNegocio;
    }

    public void setNombreNegocio(String nombreNegocio) {
        this.nombreNegocio = nombreNegocio;
    }

    public String getDescripcionNegocio() {
        return descripcionNegocio;
    }

    public void setDescripcionNegocio(String descripcionNegocio) {
        this.descripcionNegocio = descripcionNegocio;
    }

    public String getTelefonoNegocio() {
        return telefonoNegocio;
    }

    public void setTelefonoNegocio(String telefonoNegocio) {
        this.telefonoNegocio = telefonoNegocio;
    }

    public String getDireccionNegocio() {
        return direccionNegocio;
    }

    public void setDireccionNegocio(String direccionNegocio) {
        this.direccionNegocio = direccionNegocio;
    }
}

File: Python/uiMain/framesFuncionalidades/tarjetaNegocioUsuario.py
class TarjetaNegocioUsuario:
    

    def __init__(
        self, 
        nombreNegocio: str, 
        descripcionNegocio: str,
        telefonoNegocio: str,
        direccionNegocio: str) -> None:
        self._nombreNegocio = nombreNegocio
        self._descripcionNegocio = descripcionNegocio
        self._telefonoNegocio = telefonoNegocio
        self._direccionNegocio = direccionNegocio

    def getNombreNegocio(self) -> str:
        return self._nombreNegocio

    def setNombreNegocio(self, nombreNegocio: str) -> None:
        self._nombreNegocio = nombreNegocio

    def getDescripcionNegocio(self) -> str:
        return self._descripcionNegocio

    def setDescripcionNegocio(self, descripcionNegocio: str) -> None:
        self._descripcionNegocio = descripcionNegocio

    def getTelefonoNegocio(self) -> str:
        return self._telefonoNegocio

    def setTelefonoNegocio(self, telefonoNegocio: str) -> None:
        self._telefonoNegocio = telefonoNegocio

    def getDireccionNegocio(self) -> str:
        return self._direccionNegocio

    def setDireccionNegocio(self, direccionNegocio: str) -> None:
        self._direccionNegocio = direccionNegocio

    def __str__(self) -> str:
        return "TarjetaNegocioUsuario{ \n" + "nombreNegocio='" + self.getNombreNegocio() + ",\ndescripcionNegocio='" + self.getDescripcionNegocio() + ",\ntelefonoNegocio='" + self.getTelefonoNegocio() + ",\ndireccionNegocio='" + self.getDescripcionNegocio() + '\n}'

    